### PR TITLE
Revert "shell: Install and use Mitogen"

### PR DESCRIPTION
**Pull request by Zempashi**
This reverts commit 316cdb8e1a07ca64020830972b30271a627aa886.
CI with terraform is impacted by https://github.com/dw/mitogen/issues/404

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,10 +4,10 @@
 #
 #    tox -e pip-compile
 #
-ansible-modules-hashivault==3.9.8 \
-    --hash=sha256:d5a743d3fbcd5f193bccd6209f0f74b26b645accb0ea68ca70ebe73be2981424
-ansible==2.7.1 \
-    --hash=sha256:e7953472347fcc6dca10839111b576a9f790e00056344f2dcf448e6c452fe939
+ansible-modules-hashivault==3.10.1 \
```
Comment (NicolasT) [inline requirements.txt]: Huh, I'm surprised this is a *revert* which *upgrades* package versions?

**Comment by NicolasT** [comment]
@Zempashi This one should be OK to reset, right?

**Comment by Zempashi** [comment]
@bert-e reset

**Review by NicolasT — APPROVED** [review]
Hope we can get this in again soon'ish!

**Comment by Zempashi** [comment]
@bert-e approve

**Pull request merged**